apps/site/src/content/blog/astro-theme-provider.mdx

```diff
@@ -0,0 +1,84 @@
+# Introducing: Astro Theme Provider
+
+## What is Astro Theme Provider?
+
+Astro Theme Provider is a tool that allows you to author themes for Astro like a normal project and export your work as an integration for others to use. 
+
+## Problems with current theme practices
+
+In Astro, normally a theme is a full project published as a template that users can clone and customize. Project templates are a common pattern and have lots of use cases, but they also have downsides that theme authors may not always be aware of:
+
+- **Hard to upgrade**: Merging improvements from a project template can be difficult if not impossible after customizing your local copy.
+- **Incompatible with existing projects**: Project templates cannot be easily added to an existing Astro project. Adding a template to an existing project requires manuallly splicing code from the template into the project.
+- **Not beginner-friendly**: The main demographic for theme users are people who are new to Astro and web development in general. Cloning and editing a project template as a beginner can be challenging. Project templates often times use lots of different technologies and can be bloated with files beginners may not need to see or touch.
+- **Inconsistent**: There is no standard for authoring templates, most templates use different combinations of patterns and structures. For example, many templates have a config file for customization, but the location of the file and its options vary depending on the template. Learning how to use a template while also learning how to use Astro can be confusing.
```

Comment:
I would personally try to make this a paragraph, instead of bullet points, maybe try to link those sentences?